tsl/test/CMakeLists.txt

```
@@ -67,7 +67,10 @@ if (TAP_CHECKS)
     PATH="${PG_BINDIR}:$ENV{PATH}"
     PGPORT=6${TEST_PGPORT_LOCAL}
     PG_REGRESS=${PG_REGRESS}
-    ${PROVE} -I "${PG_SOURCE_DIR}/src/test/perl"
+    ${PROVE}
+    -I "${PG_SOURCE_DIR}/src/test/perl"
```

If `PG_SOURCE_DIR` is not set, this will result in `-I/src/test/perl`, which is probably not what you intended. You should probably have something like this (https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html#conditional-expressions):
```suggestion
    $<$<BOOL:${PG_SOURCE_DIR}>:-I${PG_SOURCE_DIR}/src/test/perl>
```